Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateIsotermicHumidificationProcessByRelativeHumidity.cs

```diff
@@ -0,0 +1,99 @@
+
+        protected override GH_SAMParam[] Inputs
+        {
+            get
+            {
+                List<GH_SAMParam> result = new List<GH_SAMParam>();
+                result.Add(new GH_SAMParam(new GooMollierPointParam() { Name = "_start", NickName = "_start", Description = "Start Point for MollierProcess", Access = GH_ParamAccess.item }, ParamVisibility.Binding));
+                result.Add(new GH_SAMParam(new global::Grasshopper.Kernel.Parameters.Param_Number() { Name = "_relativeHumidity", NickName = "_relativeHumidity", Description = "Relative Humidity", Access = GH_ParamAccess.item}, ParamVisibility.Binding));
+                return result.ToArray();
+            }
+        }
+
+        protected override GH_SAMParam[] Outputs
+        {
+            get
+            {
+                List<GH_SAMParam> result = new List<GH_SAMParam>();
+                result.Add(new GH_SAMParam(new GooMollierProcessParam() { Name = "isotermicHumidificationProcess", NickName = "isotermicHumidificationProcess", Description = "IsotermicHumidificationProcess", Access = GH_ParamAccess.item }, ParamVisibility.Binding));
+
+                return result.ToArray();
+            }
+        }
+
+        /// <summary>
+        /// Updates PanelTypes for AdjacencyCluster
+        /// </summary>
+        public SAMMollierCreateIsotermicHumidificationProcessByRelativeHumidity()
+          : base("SAMMollier.CreateIsotermicHumidificationProcessByRelativeHumidity", "SAMMollier.CreateIsotermicHumidificationProcessByRelativeHumidity",
+              "Creates IsotermicHumidificationProcess",
+              "SAM", "Mollier")
+        {
+        }
+
+        protected override void SolveInstance(IGH_DataAccess dataAccess)
+        {
+            int index;
+
+            index = Params.IndexOfInputParam("_start");
+            if (index == -1)
+            {
+                AddRuntimeMessage(GH_RuntimeMessageLevel.Error, "Invalid data");
+                return;
+            }
+            MollierPoint mollierPoint = null;
+            if (!dataAccess.GetData(index, ref mollierPoint) || mollierPoint == null)
+            {
+                AddRuntimeMessage(GH_RuntimeMessageLevel.Error, "Invalid data");
+                return;
+            }
+
+            index = Params.IndexOfInputParam("_relativeHumidity");
+            if (index == -1)
+            {
+                AddRuntimeMessage(GH_RuntimeMessageLevel.Error, "Invalid data");
+                return;
+            }
+            double relativeHumidity = double.NaN;
+            if (!dataAccess.GetData(index, ref relativeHumidity) || double.IsNaN(relativeHumidity))
+            {
+                AddRuntimeMessage(GH_RuntimeMessageLevel.Error, "Invalid data");
+                return;
+            }
+
+            IsotermicHumidificationProcess isotermicHumidificationProcess = Core.Mollier.Create.IsotermicHumidificationProcess_ByHumidityRatioDifference(mollierPoint, relativeHumidity);
```

Comment:
Change to RelativeHumidity Create Method

Reply:
done